docs/components/navigator.md

```diff
@@ -0,0 +1,115 @@
+<!--
+Copyright (c) 2020 The Gamepad Navigator Authors
+See the AUTHORS.md file at the top-level directory of this distribution and at
+https://github.com/fluid-lab/gamepad-navigator/raw/master/AUTHORS.md.
+
+Licensed under the BSD 3-Clause License. You may not use this file except in
+compliance with this License.
+
+You may obtain a copy of the BSD 3-Clause License at
+https://github.com/fluid-lab/gamepad-navigator/blob/master/LICENSE
+-->
+
+# gamepad.navigator
+
+This component listens to the `gamepadconnected` and `gamepaddisconnected` events in the browser. When at least one
+gamepad is connected, it reads inputs from them at a particular frequency and stores those input values in the
+component's model data. The component stops reading these inputs when the last gamepad is disconnected.
+
+## Using this grade
+
+This grade can work without passing any option to it. An instance of the `navigator` does not require any options to be
+passed and can listen to the gamepad events and read the gamepad inputs. To use it, we can create an instance of the
+component or use it as a grade another component:
+
+``` javascript
+// Either create an instance of the navigator.
+var navigatorInstanceOne = gamepad.navigator();
+
+// Otherwise create a custom component using the navigator grade.
+fluid.defaults("my.navigator.grade", {
+    gradeNames: ["gamepad.navigator"]
+};
+var navigatorInstanceTwo = my.navigator.grade();
+```
+
+## Component Options
+
+The following component configuration options are supported:
+
+| Option | Type | Default Value | Description |
+| :---: | :---: | :---: | :--- |
+| `windowObject` | `{Object}` | `window` | The global window object from which the navigator derives various methods to read gamepad inputs and attach listeners to it for various gamepad events. |
+| `frequency` | `{Integer}` | 50 | The frequency (in ms) at which the navigator will read gamepad inputs. |
+
+These options can be provided to the component in the following ways:
+
+```javascript
+// Either pass the options inside the object as an argument while creating an instance of the navigator.
+var navigatorInstanceOne = gamepad.navigator({
+    windowObject: myCustomWindowObject,
+    frequency: 100
+});
+
+// Otherwise pass it as default options in a custom component using the navigator grade.
+fluid.defaults("my.navigator.grade", {
+    gradeNames: ["gamepad.navigator"],
+    windowObject: myCustomWindowObject,
+    frequency: 100
+};
+var navigatorInstanceTwo = my.navigator.grade();
+```
+
+Though the `windowObject` is a configurable option, it should have all the properties and methods that the navigator
+uses. Otherwise, the navigator won't work and will throw errors. You can also use a mutated `window` object and pass it
```

Comment:
"modified" is probably better than "mutated", and I would give an example, such as "a modified window object, such as the mocks used in this package's tests)"